src/octopeer-github/main/Database/EventObject/KeystrokeEvent.ts

```diff
@@ -2,13 +2,19 @@
  * Created by Maarten on 26-05-2016.
  */
 
+const KeystrokeType = {
```

Comment:
I know and I double checked it, but still got it on the old commit.
Thanks for adding the suppress warning! :)